thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -275,7 +275,7 @@ export default Route.extend(AuthenticatedRouteMixin, {
         const granularity = adjustGranularity(anomalyEntity.attributes.metricGranularity[0]);
         const metricGranularity = toMetricGranularity(granularity);
 
-        const anomalyRange = [parseInt(anomalyEntity.start, 10), parseInt(anomalyEntity.end, 10) + 1];
+        const anomalyRange = [parseInt(anomalyEntity.start, 10), parseInt(anomalyEntity.end, 10)];
```

Comment:
Do you know why do we + 1 before? It would be good to leave a comment on the change. So next time people who changes this part knows the logic behind.

Reply:
In the new version, the anomalies are from 12am to 12am. The endtime exclusive should be the same for all granularity.